lib/teamsnap.rb

```diff
@@ -34,7 +34,7 @@ def init(opts = {})
       self.root_client = TeamSnap::Client.new(:token => token)
 
       ##   Make the apiv3 root call. collection is parsed JSON
-      collection = TeamSnap.run(root_client, :get, "/", {})
+      collection = TeamSnap.run(root_client, :get, self.url, {})
```

Comment:
does `Faraday.new` work with the `/v3/` path in the url? seemed like it used to have a problem with anything beyond the domain name.